tests/main/views/test_buyers.py

```diff
@@ -2127,24 +2136,151 @@ def test_do_not_show_question_and_answer_dates_for_closed_brief(self):
         assert res.status_code == 404
 
 
+@mock.patch("app.main.views.buyers.data_api_client", autospec=True)
```

Comment:
Noooooooo 🙀 Can we keep the api client patching in the setup method? This was part of the plan to move it all to a  setup method on the `BaseApplicationTest` helper 😿 😿 😿

Reply:
OH.

I wasn't aware of this and this isn't the only place I've made this transformation.